wfl/Controllers/FoodOrdersController.cs

```diff
@@ -42,5 +42,18 @@ public FoodOrdersController(IDBManager dbManager)
         {
             return Ok(await dbManager.GetEntityByIdAsync<FoodOrder>(id));
         }
+
+        [Route("{id}")]
+        [HttpPost]
+        public async Task<IHttpActionResult> SetStatusAsync([FromUri] string id, [FromBody] FoodOrderStatusDto dto)
+        {
+            FoodOrder order = await dbManager.GetEntityByIdAsync<FoodOrder>(id);
+            if (order?.Status == dto?.Status)
```

Comment:
Really like this. 👍    this allows us the flexibility to store a "comment" or "note" for why the food order status was changed. Good stuff with the usage of a `Dto` here.